packages/auth-construct/src/construct.ts

```diff
@@ -43,8 +45,9 @@ import { build as arnBuilder } from '@aws-sdk/util-arn-parser';
 
 type DefaultRoles = { auth: Role; unAuth: Role };
 type IdentityProviderSetupResult = {
-  oauthMappings: Record<string, string>;
+  oAuthMappings: Record<string, string>;
   providersList: string[];
+  oAuthSettings: cognito.OAuthSettings | undefined;
```

Comment:
One note about this change - I move oAuthSettings into the setupExternalProviders to allow for a reduction in the constructor logic.